lib/commands/context.js

```diff
@@ -184,6 +192,32 @@ helpers.startChromedriverProxy = async function startChromedriverProxy (context)
       if (androidPackage && androidPackage.length > 0) {
         opts.chromeAndroidPackage = androidPackage[1];
       }
+      if (!opts.extractChromeAndroidPackageFromContextName) {
+        if (_.has(this.opts, 'enableWebviewDetailsCollection') && !this.opts.enableWebviewDetailsCollection) {
+          // When enableWebviewDetailsCollection capability is explictly disabled, try to identify
+          // chromeAndroidPackage based on contexts, known chrome variant packages and queryAppState result
+          // since webviewsMapping does not have info object
+          const contexts = webviewsMapping.map((wm) => wm.webviewName);
+          for (const knownPackage of KNOWN_CHROME_PACKAGE_NAMES) {
+            if (_.includes(contexts, `${WEBVIEW_BASE}${knownPackage}`)) {
+              continue;
+            }
+            const appState = await this.queryAppState(knownPackage);
+            if (appState === 3 || appState === 4) {
```

Comment:
`appState >= 3` or `._includes([3, 4], appState)`

Reply:
Also it would be better to have constants for such magic numbers

Reply:
Addressed as https://github.com/appium/appium-android-driver/pull/699/commits/31925860ccf9721795242266c149243a895bfa39

Let me know if you have a concern, e.g. exporting `APP_STATE_RUNNING_IN_BACKGROUND` and `APP_STATE_RUNNING_IN_FOREGROUND`.